third-culture-software/bhima
# Set created_at dates in BHIMA test data (for System Usage Report)

Updated the BHIMA test data to set the 'created_at' times for the 'cash' table and the 'stock_movements' table.  

Since the System Usage Report relies on 'created_at' times, this makes testing that report easier with the test data (bhima_test).

This change should not affect the regression tests because none of them use the 'created_at' time.

**TESTING**
- Use bhima_test and run 'yarn build:db'
- In Hospital > Report > System Usage Statistics Report, get the report for today (the day you test it)
   - Note that there are two entries.  There should be 1 cash and 1 invoice
   - Do a stock exit. 
   - View the report again and note the stock_movement
- Run yarn:integration
   - The tests should all pass
   - Look at the System Usage Report again
     -  There should be a total of 2 patients, 3 invoices, 3 cash payments, and 11 stock movements
